services/121-service/src/main.ts

```diff
@@ -3,6 +3,7 @@ import { ApplicationModule } from './app.module';
 import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
 import { ValidationPipe } from '@nestjs/common';
 import { PORT, BASE_PATH, SCHEME } from './config';
+const appInsights = require("applicationinsights");
```

Comment:
Hmm.. why doesn't Prettier complain about these `"` instead of `'` here? Weird.

Reply:
yeah, have been fixed anyway.